### PR TITLE
[FIX WEBSITE-212] - Remove Voltron text

### PR DESCRIPTION
**Pull request by michaelmccaskill**
Related to issue [WEBSITE-212]

Summary of this pull request: 

Remove Voltron text.

full view

zoomed

### CODE REVIEW

File: app/components/Main.jsx [hunk 1]
```diff
@@ -84,7 +84,7 @@ class Main extends React.PureComponent {
           <form ref="form" action="#" id="plugin-search-form"
               className={classNames(styles.HomeHeader, { showFilter: this.props.showFilter }, 'HomeHeader jumbotron')}
               onSubmit={this.handleOnSubmit}>
-            <h1><span className="logo">project</span>Voltron</h1>
+            <h1><span className="logo">Jenkins</span>Plugins</h1>
             <p className="tagline">The strength of many, shared by all.</p>
```
Comment (daniel-beck) [inline app/components/Main.jsx:88]: This line is still weird.

**Comment by daniel-beck** [comment]
For design changes, could you include screenshots in PR description/comments?

File: app/components/Main.jsx (hunk 1 above)
Comment (michaelmccaskill) [reply app/components/Main.jsx:88]: Ok I'll kick it back to Gus and see options we can create.

**Comment by michaelmccaskill** [comment]
Sure.

**Comment by michaelmccaskill** [comment]
I added screenshots showing the current proposal but I will change them when I have an updated text.

**Comment by daniel-beck** [comment]
It looks like the current text style is tailored for the old project voltron message and looks a bit weird when it's just "Jenkins Plugins", no?

**Comment by michaelmccaskill** [comment]
Should we lose the image, too?

**Comment by rtyler** [comment]
@michaelmccaskill Nah, the voltron Jenkins is neat and fits the plugins concept nicely. :smi

**Comment by daniel-beck** [comment]
I agree with @rtyler. We've been using modified Jenkins logos a lot and they're very popular. So even if we're not calling it Voltron anymore, having a Mr Jenkinstron is fine.

**Comment by michaelmccaskill** [comment]
I'm closing this until Gus has time to come up with a new text.

**Pull request closed without merging**